### PR TITLE
Aa updates to comments and indenting

### PR DESCRIPTION
**Pull request by yodarjun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +171,28 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
     robot.add_trait(:human_like_intelligence)
     ```
 
+### RSpec Let
+* <a name="rspec-let"></a>Add new lines in `let` definitions
```
Comment (benseligman) [inline README.md]: is this necessary? we should be doing this with any `{` block that's super long

**Comment by benseligman** [comment]
:+1: overall

things i'd like to add (though these could be hashed out in a separate pull request):
- options hash < keyword arguments
- alphabetize lists of anything where order doesn't matter. so like multilined hashes or methods in a class
- always have trailing commas in multiline hashes or arrays
- the below is bad, not good. we should always indent by 2 spaces on stuff like this

`````` ruby
# good
def self.create_translation(phrase_id,
                            phrase_key,
                            target_locale,
                            value,
                            user_id,
                            do_xss_check,
                            allow_verification)
  ...
end```
``````

**Comment by michaeldauria** [comment]
:+1: I like everything @benseligman mentioned as well

File: README.md [hunk 2]
```diff
@@ -180,159 +201,31 @@ Airbnb also maintains a [JavaScript Style Guide][airbnb-javascript].
...
-
-> When writing your comments, write for your audience: the next contributor who
-> will need to understand your code. Be generous — the next one may be you!
-
-&mdash;[Google C++ Style Guide][google-c++]
-
-Portions of this section borrow heavily from the Google
-[C++][google-c++-comments] and [Python][google-python-comments] style guides.
+* Write self-documenting code and ignore the rest of this section. Seriously!
+* Avoid superfluous comments.
```
Comment (armyofgnomes) [inline README.md:228]: Sometimes "self-documenting code" gets a little ridiculous...

```
  def valid_launched_action?
    return false if self.complete?
    return_true_if_valid_launchable_action_else_return_errors == true &&
      subscription_active_or_not_required? &&
      !over_quota_for_current_monthly_period? &&
      !admin_launch_that_needs_participants?
  end
```

**Comment by benseligman** [comment]
lgtm

**Pull request closed without merging**